nityam_kumar/PostgresAssignment/src/services/classes.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import db from "../db";
+
+import AppError from "../utils/appError";
+
+export const getClasses = async () => {
+  try {
+    const data = await db.query(
+      "select distinct cl_no as class from student order by fname"
+    );
+    return data.rows;
+  } catch (err) {
+    throw new AppError(err.message, 502);
```

Comment:
will user get to know if there is a database query error or any other server error related info ???

Reply:
yes